src/pages/plugins.astro

```diff
@@ -39,12 +40,15 @@ Astro.response.headers.set("Cache-Control", "s-maxage=300");
                     <div class="plugin">
                         {/*
                         <img src={plugin.screenshot || "/assets/screenshot-placeholder.png"} class="plugin-screenshot" />
-                    */}
+                        */}
```

Comment:
decide what we want to do for this cause we need to get the devs to take screenshots if we go forward